app/pb_kits/playbook/pb_home_address_street/_home_address_street.jsx

```diff
@@ -36,6 +37,7 @@ const HomeAddressStreet = ({
   houseStyle,
   state,
   zipcode,
+  territory,
```

Comment:
Lets add `zipcode` and `territory` to the jsx template as well to keep these consistent.